src/test/java/uk/gov/pay/products/resources/ProductResourceIT.java

```diff
@@ -513,7 +515,8 @@ void findProductByGatewayAccountIdAndExternalId_shouldReturnProduct_whenFound()
                     .body(TYPE, is(product.getType().name()))
                     .body(DESCRIPTION, is(product.getDescription()))
                     .body(RETURN_URL, is(product.getReturnUrl()))
-                    .body(LANGUAGE, is("en"));
+                    .body(LANGUAGE, is("en"))
+                    .body(DATE_CREATED, is(notNullValue()));
```

Comment:
unsure if we do this elsewhere but could you inject a fixed `Clock` into this test to be able to check a specific value here (as opposed to nullness)?